src/guillo-gmi/actions/copy_item.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { PathTree } from '../components/modal'
+import { useTraversal } from '../contexts'
+import { useCrudContext } from '../hooks/useCrudContext'
+
+function getNewId(id = '') {
```

Comment:
maybe we can move this function into a utils file? The same function exists inside [`src/guillo-gmi/actions/copy_items.js`](https://github.com/guillotinaweb/guillotina_react/blob/e13eec96a79a1f3ce6e2e5175fca44713bae9123/src/guillo-gmi/actions/copy_items.js#L7-L15)

Reply:
moved @aralroca